pom.xml

```diff
@@ -232,7 +232,7 @@
     <version.org.junit>5.5.2</version.org.junit>
     <version.org.keycloak>4.8.3.Final</version.org.keycloak>
     <version.log4j>1.2.17</version.log4j>
-    <version.org.mvel>2.4.4.Final</version.org.mvel>
+    <version.org.mvel>2.4.5.Final</version.org.mvel>
```

Comment:
we already have 2.4.7.Final on master see  https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/pom.xml#L235